react/react1/week1/todolist/src/components/ItemRow.js

```diff
@@ -0,0 +1,48 @@
+import ItemDescription from "./ItemDescription";
```

Comment:
This component actually does not need to be here. You have allready done a lot of small components so why not use them directly inside the ToDoList component instead of doing this seperate one? What ends up happening is you are passing props through a lot of components and as you see this component needs to take in alot of props in order to work. This makes the code less readable and harder to debug.